bin/zos-cli.js

```diff
@@ -1,15 +1,15 @@
 #! /usr/bin/env node
 
-const Path = require('path')
-const shelljs = require('shelljs')
-
-function run() {
-  const bin = 'npx truffle exec'
-  const script = Path.resolve(__dirname, '../src/entrypoint.js')
```

Comment:
We should be able to remove the `sed` script from the prepackage script in package.json after this change, right?

Reply:
definitely!

Reply:
I addressed this comment here https://github.com/zeppelinos/zos-cli/pull/63